modules/dnn/src/tokenizer/unicode.hpp

```diff
@@ -0,0 +1,66 @@
+#pragma once
```

Comment:
Drop `#pragma once` and use `#ifdef` guard instead.

---

Need to add opencv license:

```
// This file is part of OpenCV project.
// It is subject to the license terms in the LICENSE file found in the top-level directory
// of this distribution and at http://opencv.org/license.html.
```